packages/contracts/contracts/core/permission/PermissionManager.sol

```diff
@@ -330,15 +332,16 @@ contract PermissionManager is Initializable {
     /// @param _permissionId The permission identifier required to call the method this modifier is applied to.
     function _auth(address _where, bytes32 _permissionId) private view {
         if (
-            !(isGranted(_where, msg.sender, _permissionId, msg.data) ||
-                isGranted(address(this), msg.sender, _permissionId, msg.data))
-        )
+            !isGranted(address(this), msg.sender, _permissionId, msg.data) &&
+            !isGranted(_where, msg.sender, _permissionId, msg.data)
+        ) {
             revert Unauthorized({
                 here: address(this),
                 where: _where,
                 who: msg.sender,
                 permissionId: _permissionId
             });
+        }
```

Comment:
According to the review comments I received, isn't this more our style / clearer?
```solidity
if(isGranted(address(this), msg.sender, _permissionId, msg.data)){
    return;
}

if(isGranted(_where, msg.sender, _permissionId, msg.data){
    return;
}

revert Unauthorized({
    here: address(this),
     where: _where,
     who: msg.sender,
     permissionId: _permissionId
});
```

Reply:
I don't remember that style at all. To me the current way looks cleaner. Not a big deal. As long as it's correct, let's go with it.